src/inference/hand_detection.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import cv2
+import sys
+import tensorflow as tf
+import numpy as np
+
+from yolo import YOLO
+
+
+class HandDetector:
+
+    def __init__(self):
+        self.yolo = YOLO("models/cross-hands-tiny.cfg", "models/cross-hands-tiny.weights", ["hand"])
+        self.yolo.size = int(416)
+        self.yolo.confidence = float(0.2)
+
+    def predict(self, img, draw):
```

Comment:
Add docstring about function return. I mean order of variables in each box element